frontend/public/components/_edit-yaml.scss

```diff
@@ -1,5 +1,9 @@
 .yaml-editor {
   position: relative;
+  display: flex;
+  height: 100%;
+  flex: 1;
+  flex-direction: column;
   @media (min-width: $grid-float-breakpoint) {
     // changing these values requires changes to the editorHeight() and width() calculations in edit-yaml.jsx
```

Comment:
I think this can be removed now.

Reply:
No, it's still needed. The editor height takes the min height provided if these styles are removed.

Reply:
I meant the comment :)

Reply:
Oh, sorry for the confusion. Removed.